.github/workflows/wheels-test.yaml

```diff
@@ -75,10 +75,9 @@ jobs:
           export MATRICES="
             pull-request:
               # amd64
-              - { ARCH: 'amd64', PY_VER: '3.9',  CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu18.04', gpu: 'v100', driver: 'latest' }
-              - { ARCH: 'amd64', PY_VER: '3.11', CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu22.04', gpu: 'v100', driver: 'latest' }
+              - { ARCH: 'amd64', PY_VER: '3.9',  CUDA_VER: '12.2.2', LINUX_VER: 'rockylinux8', gpu: 'v100', driver: 'latest' }
               # arm64
-              - { ARCH: 'arm64', PY_VER: '3.10', CUDA_VER: '12.0.1', LINUX_VER: 'ubuntu20.04', gpu: 'a100', driver: 'latest' }
+              - { ARCH: 'arm64', PY_VER: '3.11', CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu18.04', gpu: 'a100', driver: 'latest' }
```

Comment:
This particular combo doesn't seem to have an `arm` variant: https://hub.docker.com/layers/rapidsai/citestwheel/cuda11.8.0-ubuntu18.04-py3.11/images/sha256-71f5576469cfa14dce97e3fb4d8e7aec7c75ee3f2136396d11909f075e0a84a8?context=explore